autowrap/Utils.py

```diff
@@ -83,6 +83,8 @@ def compile_and_import(name, source_files, include_dirs=None, **kws):
     else:
         compile_args = ""
 
+    compile_args += ", '-std=c++11' "
```

Comment:
I think this needs to be a parameter, I dont think we can simply add this to all code...

Reply:
Okay, what should the default be c++03 or c++11?

Reply:
I would say default should be C++03 to make this backwards compatible and act the same way after an upgrade

Reply:
Yes, before we merge this should be fixed. Please introduce an extra flag argument for this function and also check on windows. On Win this modification will result in `compile_args = ", '-std=c++11'"` which might break the generated `setup.py` file.
I recommend to collect the compile args as a list of strings and finally use `", ".join(...)` to create the final string.

Reply:
just out of curiosity: what config/platform does not support c++11 these days?

Reply:
We don't know. C++11 is also not guaranteed to be downwards compatible (https://stackoverflow.com/questions/6399615)

Reply:
Okay, I've added a check in compile_and_import to check for which std it is passed. I've added options for c++11 14 17 and 20

Reply:
@uweschmitt looking back at the original code, it looks like I was incorrect and this isn't necessary. Are you okay with the current revision which just removes it?

Reply:
👍